llvm/lib/Target/LoongArch/LoongArch.td

```diff
@@ -118,6 +118,12 @@ def FeatureLAM_BH
                         "Support amswap[_db].{b/h} and amadd[_db].{b/h} instructions.">;
 def HasLAM_BH : Predicate<"Subtarget->hasLAM_BH()">;
 
+// Assume div.w[u] and mod.w[u] can handle inputs that are not sign-extended.
+def FeatureDiv32
+    : SubtargetFeature<"div32", "HasDiv32", "true",
+                        "Support div.w[u] and mod.w[u] can handle inputs that are not sign-extended">;
```

Comment:
Clang is not the only LLVM frontend, and the 3A5000 is still not outdated, so I recommend setting the `div32` feature to `false` by default.

Reply:
The `true` here is not the default value. It means when we have feature `div32`, then set `HasDiv32` to `true`.

But the test doesn't show how this feature work. I think it need some changes.

Reply:
New commit has added the test.